content/docs/pipelines/overview/concepts/run.md

```diff
@@ -11,9 +11,9 @@ details page on the Kubeflow Pipelines UI, where you can see the runtime graph,
 output artifacts, and logs for each step in the run.
 
 <a id=recurring-run> 
-A *recurring run* is a repeatable run of a pipeline. The
-configuration for a recurring run includes a copy of a pipeline with all
-parameter values specified and a 
+A *recurring run* (also refered to as a job in the API) is a repeatable run of
```

Comment:
I would recommend clarifying which API this refers to and adding a link. Also, I think it may be possible to shorten the sentence a little bit. How about something like this:

A *recurring run*, or [*job*](/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/#tag-JobService) in the Kubeflow Pipelines API, is a repeatable run of a pipeline.

Reply:
Awesome! love your recommendation, changing accordingly.